packages/verkle/src/types.ts

```diff
@@ -11,11 +11,11 @@ export interface VerkleTreeOpts {
   /**
    * An instantiated Verkle Cryptography interface
    */
-  verkleCrypto: VerkleCrypto | undefined
+  verkleCrypto?: VerkleCrypto | undefined
   /**
    * A database instance.
    */
-  db: DB<Uint8Array, Uint8Array>
+  db?: DB<Uint8Array, Uint8Array>
```

Comment:
So a thought here now that I see the code, we should either make these two opts mandatory or else replace the `createRootNode` with a general `init` that requires the db and verkle crypto objects and then initializes the tree with this empty root node.  In doing so, we should also check to see if the tree has the `useRootPersistence` opt set and throw if so and the rootnode already exists (since that would overwrite the root node and be destructive to the existing trie).

To do this, you need to do `tree.findPath(getVerkleStem(this.root))` and see if you get back an existing node.

What do you think?